projects/angular/src/data/datagrid/built-in/filters/datagrid-numeric-filter.ts

```diff
@@ -23,6 +23,7 @@ import { DatagridNumericFilterImpl } from './datagrid-numeric-filter-impl';
   providers: [{ provide: CustomFilter, useExisting: DatagridNumericFilter }],
   template: `
     <clr-dg-filter [clrDgFilter]="registered" [(clrDgFilterOpen)]="open">
+      <h6 aria-description="filter type">Numeric filter</h6>
```

Comment:
This cannot be an `h6` because it might not be valid in the page that contains the datagrid.

Reply:
Also, remember translation. We must use `commonStrings`.